docs/content/any/project/changelogs/2021-09-29.md

```diff
@@ -118,19 +118,18 @@ oso.registerClass(User, { fields: { ... } });
 Before this change only methods defined on value receivers could be called.
 
 ```go
-type Foo struct {
+type Vertex struct {
   X, Y float64
 }
 
 // This could be called from Polar before
-func (v Vertex) Abs() float64 {
+func (v Vertex) Sum() float64 {
 	return math.Sqrt(v.X*v.X + v.Y*v.Y)
```

Comment:
Out of curiosity, what does it mean to calculate the `Sum` of a `Vertex`?

Reply:
guess I should rename `Vertex` to `ACoupleANumbahs`